CMakeLists.txt

```
@@ -10,7 +10,7 @@ set(CVC4_MINOR   7) # Minor component of the version of CVC4.
 set(CVC4_RELEASE 0) # Release component of the version of CVC4.
 
 # Extraversion component of the version of CVC4.
-set(CVC4_EXTRAVERSION "-prerelease")
+set(CVC4_EXTRAVERSION "-release")
```

Releases are always without `CVC4_EXTRAVERSION`

```suggestion
set(CVC4_EXTRAVERSION "")
```